src/app/parcels/add/formSections/TimeofDayCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { CardProps, onChangeDateOrTime, errorText } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+import StyledTimePicker from "@/components/DataInput/TimePicker";
+import { ErrorText } from "@/components/Form/formStyling";
+
+const TimeOfDayCard: React.FC<CardProps> = ({ fieldSetter, formErrors, errorSetter }) => {
+    const [timePickerValue, setTimePickerValue] = React.useState<Date | null>(null);
+    return (
+        <GenericFormCard
+            title="Packing Time"
+            required={true}
+            text="What date is the parcel due to be packed?"
+        >
+            <>
+                <StyledTimePicker
+                    onChange={(value: any) => {
+                        const newValue = value as Date | null;
```

Comment:
if we use `unknown` as the type for `value`, I think we can avoid using type casting and just do an `if (value instanceof Date | null)`

Reply:
Mentioned above, don't think it is necessary.

Reply:
Nevermind typescript doesn't like it as technically it isn't just a date, there is also context (which I don't fully understand but it messes w types)